modules/core/src/main/java/org/apache/ignite/internal/commandline/cache/CacheArguments.java

```diff
@@ -71,6 +72,23 @@
     /** Full config flag. */
     private boolean fullConfig;
 
+    /** Cache kind. */
+    private CacheKind cacheKind = CacheKind.ALL;
+
+    /**
+     *
+     */
+    public CacheKind getCacheKind() {
```

Comment:
You should use single line comment, for empty javadoc `/** */`

Reply:
+1. Please add obvious javadoc. `@return Cache kind.`

Reply:
Done.